detekt-rules-style/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/UnusedImportSpec.kt

```diff
@@ -898,4 +898,24 @@ class UnusedImportSpec(
 
         assertThat(subject.lintWithContext(env, mainFile, compile = false)).hasSize(2)
     }
+
+    @Test
+    fun `does not report used inline val import`() {
+        val mainFile =
+            """
+            import additional.myVal
+            
+            fun main() {
+                println(myVal)
+            }
+            """.trimIndent()
+        val additionalFile =
+            """
+            package additional
+
+            inline val myVal get() = 1
+            """.trimIndent()
+
+        assertThat(subject.lintWithContext(env, mainFile, additionalFile, compile = true)).isEmpty()
```

Comment:
```suggestion
        assertThat(subject.lintWithContext(env, mainFile, additionalFile, compile = false)).isEmpty()
```
I'm not sure why all these tests have compile = false but they do. And this pr isn't passing exactly for that so no need to add the compile = true.